### DISCUSSION TITLE
AI content filter

### DISCUSSION THREAD

**Discussion by Launcherspider** [ReVanced/revanced-discussions#1092] [General]
It would be nice if there was a way to mark a video as containing ai-generated voice and/ or scripts.
There are rumors that YouTube is planning to implement this themselves. But without user input it will be ineffective. 
I recently watched to different videos, which are indistinguishable from actual "human" ones. After 10 minutes of incoherent rambling I was left with a very unpleasant feeling

**Comment by oSumAtrIX** [comment]
You can dislike videos and mark them as not interested.
